iothub/device/tests/Authentication/ClientAuthenticationWithTokenRefreshTests.cs

```diff
@@ -240,6 +240,47 @@ public async Task ClientAuthenticationWithSakRefresh_WithModule_SharedAccessKeyC
             Assert.AreEqual(token1, token2.TokenValue);
         }
 
+        [TestMethod]
+        [ExpectedException(typeof(FormatException))]
+        public void ClientAuthenticationWithSakRefresh_InvalidKey_Throws()
+        {
+            var builder = new SharedAccessSignatureBuilder()
+            {
+                Key = TestSharedAccessKey + "123",
+            };
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(FormatException))]
+        public void ClientAuthenticationWithSakRefresh_InvalidKey1_Throws()
```

Comment:
What makes this one invalid too?

Reply:
removing the test since the key is generated at run time and replacing char 'Y' wont work always.